### ISSUE TITLE
PollRawEvent and ParseRawEvent not implemented on Windows

### ISSUE THREAD

**Issue by japanoise** [nsf/termbox-go#165]
```
>go get github.com/japanoise/gomacs 
# github.com/japanoise/termbox-util 
go\src\github.com\japanoise\termbox-util\input.go:18: undefined: termbox.PollRawEvent 
go\src\github.com\japanoise\termbox-util\input.go:19: undefined: termbox.ParseEvent
```

**Comment by nsf** [comment]
That is true. Even when it will be there, I was thinking about naming it in a different way. To emphasize the fact that this API is not portable. I don't know when I'm going to implement it. On windows it's supposed to give access to https://msdn.microsoft.com/en-us/library/windows/desktop/ms683499(v=vs.85).aspx. Directly. Which is far from how it works on linux.

**Comment by japanoise** [comment]
With this in mind, I've put a workaround in my code, since if this is implemented in a different enough way, it likely won't be what I want.
